packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -85,33 +85,65 @@ const defaultProps = {
   panelSize: 'small',
 };
 
+/**
+ * This method ensures that aria-expanded attribute is allowed with the node role to avoid accessibility violations.
+ * More info on associated roles can be found here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#associated_roles
+ * @param {HTMLElement} node - a node to check
+ * @returns true if node's role is compatible with aria-expanded attribute
+ */
+const roleIsCompartible = (node) => {
```

Comment:
What does compartible mean?

Reply:
It checks if node's role is one compatible (sorry for typo, will fix if we won't remove it at the end) with aria-expanded attribute. Here is the documentation for [roles associated with aria-expanded attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#associated_roles).

Reply:
Oh, it is just a typo :).  I still don't think we need a method like this though.

Reply:
Removed